client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -199,20 +196,17 @@ const PlanOnlyThankYou = ( {
 
 	const siteUrl = useSelector( ( state ) => getSiteUrl( state, siteId as number ) );
 
-	// Display the progress bar if an ecommerce site is still being transferred
-	if ( isWpComEcommercePlan( primaryPurchase.productSlug ) && transferInProgress ) {
-		return <TransferPending orderId={ receiptId as number } siteId={ siteId as number } />;
-	}
-
 	return (
 		<>
-			{ /* If an ecommerce site is verified and completely transferred,
-			     automatically log the user into the wp-admin. */ }
```

Comment:
I also moved the comment to the code block, so we don't create a new block just for the comment.